apps/apps.humanatlas.io/src/app/app.component.scss

```diff
@@ -11,3 +11,11 @@
     flex-direction: column;
   }
 }
+
+@media only screen and (max-width: 944px) {
```

Comment:
You should be able to remove this. The content already spans the full with and doesn't have any margin/padding